src/js/components/Anchor/__tests__/__snapshots__/Anchor-test.js.snap

```diff
@@ -16,7 +15,7 @@ exports[`Anchor disabled renders 1`] = `
   box-sizing: border-box;
   font-size: inherit;
   line-height: inherit;
-  color: #6194EB;
```

Comment:
why are we changing this?

Reply:
The previous color wasn't accessible. I've discussed with Chris and he's going to work on some new colors. So, until those show up, these changes are at least accessible.